starcounter-include.html

```diff
@@ -203,24 +203,29 @@
          * @param  {String} path Polymer notification path
          * @param  {Mixed} value New value
          */
-        LauncherPartialPrototype._notifyPath = function(path, value) {
+        StarcounterIncludePrototype._notifyPath = function(path, value) {
```

Comment:
I wanted to avoid regexps and replaces if they are not needed, not to affect performance of all data-bindings (we call that on every single change that happens within this partial)

All of that will get simplified just afterhttps://github.com/Starcounter/starcounter-include/issues/38

However we should do that after line 207, as we don't want to react on changes of `model.` (see https://github.com/Starcounter/RebelsLounge/issues/85)

I don't think we should compromise performance here.